Denis_Keldanovich/06.AnsibleWorkshop/config/nginx.conf

```diff
@@ -0,0 +1,27 @@
+server {
```

Comment:
Please make templates, instead files, there will be variable virtual_host as minimum